app/models/cms/layout.rb

```diff
@@ -21,22 +21,18 @@ def content_type
   def to_xml(options = {})
     xml = options[:builder] || Nokogiri::XML::Builder.new
 
-    xml.layout do |x|
+    xml.__send__(CMS::XML::CLASS_TAG.call(self.class)) do |x|
       unless new_record?
         xml.id id
         xml.created_at created_at.xmlschema
         xml.updated_at updated_at.xmlschema
       end
-
+      x.title title
       x.system_name system_name
-      x.content_type content_type
-      x.handler handler
```

Comment:
did we discuss these two also?

Reply:
Same, removed for being fixed, empty or not applicable fields for this type.